### PR TITLE
labels

### PR DESCRIPTION
**Pull request by kenlog**
I propose the labels aligned in the center for consistency

### CODE REVIEW

**Comment by aljawaid** [comment]
I purposely left them on the left to be a bit unique but cool if you want to change them.

**Comment by creecros** [comment]
I'd prefer them even with the input boxes, and would most prefer then gone and moved as placeholders in the input boxes. The asterisks are not needed, just uneccessary eye sores, we all know they are required.

**Comment by creecros** [comment]
Also, I need help, if either of you are inerested. Check out the test branch, and see what I got going on. I have no idea which css I should be using, and grouping to create color schemes.

**Comment by creecros** [comment]
Also, all this styling on the login panel could be options, so everyone can have what they prefer.

**Comment by aljawaid** [comment]
@kenlog it's cool, I don't mind centering them.

@creecros will work on the input boxes tonight plus the asterisks, I agree with you.

Test branch.... I was looking at your commits last night and thought what's going on! haha  If you want to create pre-set colour themes then I would suggest create a setting in the plugin and then wrap your code:

if (plugin-theme) = customier-blue _apply_ customizer-blue.css otherwise `customizer.css`

So basically some sort of multiple IF statements which fallback on the standard css sheet.

In terms of the actual css, I would suggest, if you can get the css file routing sorted out, then the classes used in the standard stylesheet can be duplicated for the new colour themes.  Moving forward, they would all be maintained together as they would be just colour differences.

**Comment by creecros** [comment]
Test branch:

Ok, so let me explain what's actually going on. It's not a pre-set color scheme but instead a customizer for a color scheme.

Change "Use Custom Colors" to Yes

image

and then these options appear:

image

the idea is to allow users to create their own color shemes. I somewhat succeeded, but not so good. Here is an example:

image

This can't be in a CSS file, it has to be in a PHP file, otherwise I can't pass variables into it, so the css file is here: https://github.com/creecros/Customizer/blob/creecros-test-1/Template/layout/css.php

**Comment by creecros** [comment]
Oh, will need to have default values, otherwise the first time it is turned on everything will be white:
https://github.com/creecros/Customizer/blob/9d17f3c4772902c94531918740d105bc3d63928d/Plugin.php#L26-L33

**Comment by aljawaid** [comment]
> This is also the if statement applying it, so if 'yes' is selected it attaches the css style in the php file to the hook in layout.php, otherwise it simply does not attach it, and uses the normal css.

Is it sorted then?

**Comment by creecros** [comment]
Does it work? Yes. Looking to improve it though.

**Comment by aljawaid** [comment]
The asterisk, alignment etc changes are done, just trying to figure out the placeholder issue #17 will push soon if I can't find a way.

**Comment by aljawaid** [comment]
#18
